core/src/main/java/com/novoda/espresso/ViewTestRule.java

```diff
@@ -1,50 +1,65 @@
 package com.novoda.espresso;
 
 import android.app.Instrumentation;
-import android.content.Intent;
 import android.support.annotation.LayoutRes;
 import android.support.test.InstrumentationRegistry;
 import android.support.test.rule.ActivityTestRule;
 import android.view.View;
+import android.view.ViewGroup;
 
-public class ViewTestRule<T extends View> extends ActivityTestRule<ViewActivity> {
+import static android.view.ViewGroup.LayoutParams.MATCH_PARENT;
+
+public class ViewTestRule<T extends View> extends ActivityTestRule<EmptyActivity> {
 
     private final Instrumentation instrumentation;
+    private final ViewCreator<T> viewCreator;
+
+    private T view;
+
+    public ViewTestRule(@LayoutRes int layoutId) {
+        this(new InflateFromXmlViewCreator<T>(layoutId));
+    }
 
-    @LayoutRes
-    private final int id;
+    public ViewTestRule(ViewCreator<T> viewCreator) {
+        this(InstrumentationRegistry.getInstrumentation(), viewCreator);
+    }
 
-    public ViewTestRule(@LayoutRes int id) {
-        super(ViewActivity.class);
-        this.id = id;
-        instrumentation = InstrumentationRegistry.getInstrumentation();
+    public ViewTestRule(Instrumentation instrumentation, ViewCreator<T> viewCreator) {
+        super(EmptyActivity.class);
+        this.instrumentation = instrumentation;
+        this.viewCreator = viewCreator;
     }
 
     @Override
-    protected Intent getActivityIntent() {
-        Intent intent = new Intent(Intent.ACTION_MAIN);
-        intent.putExtra(ViewActivity.EXTRA_LAYOUT_ID, id);
-        return intent;
+    protected void afterActivityLaunched() {
+        super.afterActivityLaunched();
+        final EmptyActivity activity = getActivity();
+        view = viewCreator.createView(activity, (ViewGroup) activity.findViewById(android.R.id.content));
```

Comment:
passing the `viewParent` though it's not strictly necessary. There should be a single view being represented in this activity, that will be matchParent matchParent.

Maybe that's too strict and we should not set the layoutParams explicitly since the `view` will have to have some on it.